recipes/jacusa2/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: jacusa2
+  version: "2.0.4"
+
+source:
+  # Directly download the pre-built jar from GitHub releases
+  url: https://github.com/dieterich-lab/JACUSA2/releases/download/v2.0.4/JACUSA_v2.0.4.jar
```

Comment:
can you template the version number please?

Reply:
done, thank you